src/test/java/com/microsoft/sqlserver/jdbc/unit/statement/StatementTest.java

```diff
@@ -1866,7 +1866,7 @@ public void testNBCROWWithRandomAccess() throws Exception {
                 con = ds.getConnection();
                 Statement stmt = con.createStatement();
```

Comment:
Has this class been worked upon too? I see lot of connection/statements not closed properly.
Also `Class.forName("com.microsoft.sqlserver.jdbc.SQLServerDriver");` is unnecessary inside tests.. let's remove them too.